README.md

```diff
@@ -50,6 +54,42 @@ git checkout stable
 cd ..
 ```
 
+## Setup Instructions for Windows 11 with Visual Studio 2022
+
+1. **Install Visual Studio 2022**: Download and install from [Microsoft Visual Studio](https://visualstudio.microsoft.com/vs/). If previously installed, navigate to the "Tools" tab and select "Get tools and features".
+2. During setup, select the "Desktop Development with C++" workload. Ensure the following components are installed:
+   - MSVC v143 - VS 2022 C++ x64/x86 build tools
+   - C++ ATL for latest 143 build tools
+   - Security Issue Analysis
+   - C++ Build Insights
+   - Just-In-Time debugger
+   - C++ profiling tools
+   - C++ CMake tools for Windows
+   - Test Adapter for Boost.Test
+   - Test Adapter for Google Test
+   - Live Share
+   - C++ AddressSanitizer
+   - Windows 11 SDK
+   - vcpkg manager
+   - GitHub Copilot (optional)
+3. Additionally, navigate to the “Individual components” tab and install the "MSVC v141 - VS 2017 C++ x64/x86 build tools (v14.16)".
+4. **Restart** your computer once everything is installed.
+
+### Building the Project
+
+1. Open the `EmotiBitOscilloscope` Visual Studio solution file (.sln) located in `of_v0.11.2_vs2017_release\addons\ofxEmotiBit\EmotiBitOscilloscope`.
+2. If prompted to install extra components in the solution explorer menu, click on the install button to proceed.
+3. Build and run the solution file in the debug profile once all components have been installed. If the build fails, retarget the solution to the latest version (10.0 Windows SDK version, v143 build tools).
+
+### Notes
```

Comment:
Should this be added to the 2nd bullet point in the `setup instructions` above?

Reply:
no because this can only be done once visual studio is installed. sometimes visual studio would ask to install extra components once you open a solution file and try to run it